optimism/src/mips/interpreter.rs

```diff
@@ -18,66 +18,6 @@ pub const SYSCALL_READ: u32 = 4003;
 pub const SYSCALL_WRITE: u32 = 4004;
 pub const SYSCALL_FCNTL: u32 = 4055;
 
-// Source: https://www.doc.ic.ac.uk/lab/secondyear/spim/node10.html
-// Reserved for assembler
-pub const REGISTER_AT: u32 = 1;
```

Comment:
But wouldn't this complicate readability whenever the interpreter has to use them? I find it clearer if these arbitrary numbers have a constant name that I can relate to, rather than just the plain values.

Reply:
The names don't help with readability, they're just actively confusing. These names are how the user should think about them; we're not the user.